Examples/ObjectiveC/ExtrusionsExample.m

```diff
@@ -0,0 +1,56 @@
+//
+//  ExtrusionsExample.m
+//  Examples
+//
+//  Created by Jordan Kiley on 5/11/17.
+//  Copyright © 2017 Mapbox. All rights reserved.
+//
+
+#import "ExtrusionsExample.h"
+
+@import Mapbox;
+
+NSString *const MBXExample3DExtrusions = @"ExtrusionsExample";
+
+@interface ExtrusionsExample () <MGLMapViewDelegate>
+
+@end
+
+@implementation ExtrusionsExample
+
+- (void)viewDidLoad {
+    [super viewDidLoad];
+    
+    MGLMapView *mapView = [[MGLMapView alloc] initWithFrame:self.view.bounds styleURL:[MGLStyle lightStyleURLWithVersion:9]];
+    
+    // Center the map on the Colosseum in Rome, Italy.
+    [mapView setCenterCoordinate:CLLocationCoordinate2DMake(41.8902, 12.4922)];
+    
+    // Set the map view camera's pitch and distance.
+    mapView.camera = [MGLMapCamera cameraLookingAtCenterCoordinate:mapView.centerCoordinate fromDistance:600 pitch:60 heading:0];
+    mapView.delegate = self;
+    
+    [self.view addSubview:mapView];
+}
+
+- (void)mapView:(MGLMapView *)mapView didFinishLoadingStyle:(MGLStyle *)style {
+    
+    // Access the Mapbox Streets source and use it to create a `MGLFillExtrusionStyleLayer`. The source identifier is `composite`. Use the `sources` property on a style to verify source identifiers.
```

Comment:
This comment refers to properties in the runtime styling API. There are so many differences between that API and the style specification’s documentation on [sources](https://www.mapbox.com/mapbox-gl-js/style-spec/#root-sources) that we’d want to link to the more detailed [API reference](https://www.mapbox.com/ios-sdk/api/3.5.4/Classes/MGLStyle.html) instead.

In any case, I think https://github.com/mapbox/ios-sdk-examples/pull/86#discussion_r116355291 is relevant here as well:

> If you aren’t sure what the `source` is called, you can view the style in Mapbox Studio or iterate over the style’s `sources` property, printing out each source’s identifier.